src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -52,8 +52,6 @@ spec:
           value: {{ cluster_cfg['rest-server']['default-pai-admin-password'] }}
         - name: ETCD_URI
           value: {{ cluster_cfg['rest-server']['etcd-uris'] }}
-        - name: NV_DRIVER
-          value: /var/drivers/nvidia/current
```

Comment:
for example if on aks env dir is diff with this, need tell user to  change this config?

Reply:
No, this can be hardcoded. My plan is all gpu-consumers(job-exporter, node-manager) use this symbolic link to find current active driver. And it's driver's responsibility to make this symbolic link pointing to the right place, whether it is pre-installed place or our installation place.